pom.xml

```diff
@@ -18,9 +18,9 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>com.opentable</groupId>
-    <artifactId>otj-parent-spring</artifactId>
-    <version>328</version>
+    <groupId>org.basepom</groupId>
+    <artifactId>basepom-minimal</artifactId>
```

Comment:
This is the heart of it, and everything else is mostly just copying deps, and profiles.